openapi/paths/news@{id}.yaml

```diff
@@ -24,6 +24,64 @@ get:
                         $ref: ../components/schemas/news/body.yaml
                       author:
                         $ref: ../components/schemas/news/author.yaml
+                      attachedFiles:
+                        description: Files attached to this article.
+                        type: array
+                        items:
+                          type: object
```

Comment:
Is there an object definition for this.. It seems weird we need to repeat ourselves like this to describe a standard file object..

Reply:
There is, but I thought it's different because the definition didn't have "type".
Actually, it was the other way around. I've added type to file.yaml and changed this to refer to it.
I've created master PR to add type to file.yaml.